core/src/main/java/com/google/errorprone/bugpatterns/StringCaseLocaleUsage.java

```diff
@@ -0,0 +1,73 @@
+package com.google.errorprone.bugpatterns;
```

Comment:
We'll need the copyright headers, I can also add them during the import if that's easier.

```
/*
 * Copyright 2023 The Error Prone Authors.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

('The Error Prone Authors' refers to https://github.com/google/error-prone/blob/master/AUTHORS, you're welcome to add yourself there also. There's some more detail about that in https://g3doc.corp.google.com/company/teams/opensource/releasing/authors.md?cl=head)

Reply:
I will push a commit with the copyright headers.

> you're welcome to add yourself there also.

That'd be cool! Added that in the commit.

I can't reach the website with more details though, it asked me to do a sign in with a "@google.com" email, so it's probably an internal thing?

Reply:
Sorry, there's an external mirror with the same content, I just pasted the wrong link. That should have been https://opensource.google/documentation/reference/releasing/authors